website/pages/docs/cheatsheet.mdx

```diff
@@ -25,6 +25,10 @@ Built Sierra code of this package will be written to `target/dev` directory.
 
 <Callout>`scarb clean{:shell}` cleans `target` directory.</Callout>
 
+Building also caches required dependencies in [cache directory](https://docs.swmansion.com/scarb/docs/reference/global-directories#cache-directory).
+
+<Callout>`scarb cache clean{:shell}` removes cache directory.</Callout>
```

Comment:
I don't think this is essential enough for the cheatsheet.
Maybe a note in `global-directories#cache-directory` would suffice? 
CC: @mkaput